ashley/build.gradle

```diff
@@ -0,0 +1,32 @@
+buildscript {
+  ext {
+    junitPlatformVersion = "1.0.0-M4"
+  }
+  dependencies {
+    classpath "org.junit.platform:junit-platform-gradle-plugin:$junitPlatformVersion"
```

Comment:
Move classpath dependencies to root `build.gradle` file. Remove `buildscript` block.